frontend/src/component/admin/apiToken/CreateApiToken/CreateApiToken.tsx

```diff
@@ -107,7 +105,7 @@ export const CreateApiToken = ({ modal = false }: ICreateApiTokenProps) => {
                 handleSubmit={handleSubmit}
                 handleCancel={handleCancel}
                 mode="Create"
-                actions={<CreateButton name="token" permission={permission} />}
+                actions={<CreateButton name="token" permission={ADMIN} />}
```

Comment:
All types of tokens currently. It's essentially the same behavior as before: Only `ADMIN`s are able to create API tokens, except now we're being specific about it instead of pointing to a permission that only `ADMIN`s have anyways.

Of course with the new permissions and custom root roles you're now able to create tokens of a specific type if you have that permission, but only on the API level, it's not reflected on the UI yet.

This is what I meant in:
>There's some UI permission logic around this, but in the future https://linear.app/unleash/issue/2-1156/adapt-api-tokens-creation-ui-to-new-permissions could take it a bit further by adapting the creation of tokens as well.

Reply:
> It's essentially the same behavior as before: Only ADMINs are able to create API tokens, except now we're being specific about it instead of pointing to a permission that only ADMINs have anyways.

Thanks! Cristal clear!